pages/__tests__/index.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { theaters } from 'definitions/theater'
+
+import Index from '../index'
+
+test('menu exists', () => {
+  const { getByText } = render(<Index />)
+  theaters.forEach(theater => {
+    expect(getByText(`Theater ${theater}`))
+  })
+})
```

Comment:
```suggestion
import React from 'react'
import { render, screen } from '@testing-library/react'
import { theaters } from 'definitions/theater'

import Index from '../index'

test('menu exists', () => {
  render(<Index />)
  theaters.forEach(theater => {
    expect(screen.getByText(`Theater ${theater}`))
  })
})
```

This is the new recommendation 👍